java/src/main/java/ai/rapids/cudf/ReductionAggregation.java

```diff
@@ -256,7 +256,7 @@ public static ReductionAggregation collectSet() {
    * @param nanEquality  Flag to specify whether NaN values in floating point column should be considered equal.
    */
   public static ReductionAggregation collectSet(NullPolicy nullPolicy,
-      NullEquality nullEquality, NaNEquality nanEquality) {
+                                                NullEquality nullEquality, NaNEquality nanEquality) {
```

Comment:
Is this style adjustment necessary?

Reply:
I think so. This is what the IDE automatically reformat. I checked other files and this seems consistent. For example: https://github.com/rapidsai/cudf/blob/a6b969993fe7afc6db21c626a0b9d241e55afb95/java/src/main/java/ai/rapids/cudf/ColumnVector.java#L151-L153